analysis/tests/src/expected/Completion.res.txt

```diff
@@ -1518,3 +1518,27 @@ Completable: Cpath array->ma
     "documentation": null
   }]
 
+Complete src/Completion.res 397:14
+posCursor:[397:14] posNoWhite:[397:13] Found expr:[396:14->397:20]
+Pexp_apply ...__ghost__[0:-1->0:-1] (...[396:14->397:16], ...[397:16->397:19])
+posCursor:[397:14] posNoWhite:[397:13] Found expr:[396:14->397:16]
```

Comment:
This `[396:14->397:16]` should cover the location of `red` but only covers `ed`.